src/main/daml/Daml/Finance/Interface/Common/Types.daml

```diff
@@ -50,7 +65,7 @@ data InstrumentKey = InstrumentKey
       -- ^ Party providing depository services.
     issuer : Party
       -- ^ Issuer of instrument.
-    id : Id
+    id : VersionedId
```

Comment:
similarly here: we would allow two instruments with the same label to co-exist. I think for the uniqueness, only `label` and `version` should matter, so maybe inline those explicitly here?

Reply:
Will inline it as above

Reply:
As per the `Account`, I've created a dedicated `InstrumentId` and added a separate `description` field to the base instrument interface.